src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx

```diff
@@ -15,6 +15,7 @@ interface CanvasSlideshowProps {
   containerWidth: number
   disclaimer: any
   unit: any
+  renderTitleCard?: boolean
```

Comment:
it is not clear to me where the renderTitleCard prop is being used

Reply:
It's being used here: https://github.com/damassi/reaction/blob/master/src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx#L151

Reply:
Does it not get redefined here? https://github.com/damassi/reaction/blob/master/src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx#L143

Reply:
but i think it is redefined just above https://github.com/damassi/reaction/blob/master/src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx#L144

Reply:
Ah, you are correct -- will fix.